package.json

```diff
@@ -1,29 +1,43 @@
 {
-  "name": "gt-scheduler",
+  "name": "gt-scheduler-v2",
   "version": "0.1.0",
   "private": true,
-  "homepage": "https://jasonpark.me/gt-scheduler",
+  "homepage": "https://gtscheduler.tech/",
```

Comment:
Revert `name` and `homepage`, and increase version to `0.2.0`. Please refer to [Semantic Versioning](https://semver.org) how a version number is determined.